pkg/scheduler/actions/common/allocate.go

```diff
@@ -27,10 +27,26 @@ func AllocateJob(ssn *framework.Session, stmt *framework.Statement, nodes []*nod
 		return false
 	}
 
+	nodeSets, err := ssn.SubsetNodesFn(job, !isPipelineOnly, nodes)
+	if err != nil {
+		log.InfraLogger.V(6).Infof(
+			"Failed to run SubsetNodes on job <%s/%s>", job.Namespace, job.Namespace)
+		return false
```

Comment:
Since it is going to reject workload scheduling, it should be clearly logged with error logger so it will be visible regardless of currently set log level:
```
log.InfraLogger.Errorf
```